### PR TITLE
[Xcode13] Bump for Xcode 13 beta4.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tests/xtro-sharpie/MacCatalyst-UIKit.todo [hunk 1]
```diff
@@ -1193,3 +1190,4 @@
 !missing-type! UIWindowSceneActivationInteraction not bound
 !missing-type! UIWindowSceneActivationRequestOptions not bound
 ## appended from unclassified file
+## appended from unclassified file
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/MacCatalyst-UIKit.todo:1193]:
```suggestion
```

File: tests/xtro-sharpie/MacCatalyst-Vision.todo [hunk 2]
```diff
@@ -57,3 +74,4 @@
 !missing-type! VNDetectDocumentSegmentationRequest not bound
 !missing-type! VNGeneratePersonSegmentationRequest not bound
 !missing-type! VNHumanObservation not bound
+## appended from unclassified file
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/MacCatalyst-Vision.todo:77]:
```suggestion
```

File: tests/xtro-sharpie/MacCatalyst-WebKit.todo [hunk 3]
```diff
@@ -13,19 +14,19 @@
...
-!missing-selector! WKWebView::pauseAllMediaPlayback: not bound
-!missing-selector! WKWebView::requestMediaPlaybackState: not bound
-!missing-selector! WKWebView::resumeAllMediaPlayback: not bound
 !missing-selector! WKWebView::setUnderPageBackgroundColor: not bound
 !missing-selector! WKWebView::suspendAllMediaPlayback: not bound
+!missing-selector! WKWebView::themeColor not bound
 !missing-selector! WKWebView::underPageBackgroundColor not bound
+!missing-selector! WKWebViewConfiguration::setUpgradeKnownHostsToHTTPS: not bound
+!missing-selector! WKWebViewConfiguration::upgradeKnownHostsToHTTPS not bound
+## appended from unclassified file
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/MacCatalyst-WebKit.todo:32]:
```suggestion
```

File: tests/xtro-sharpie/iOS-CoreVideo.todo [hunk 4]
```diff
@@ -1,4 +1,7 @@
 !deprecated-attribute-missing! CVBufferGetAttachment missing a [Deprecated] attribute
 !deprecated-attribute-missing! CVBufferGetAttachments missing a [Deprecated] attribute
+!missing-field! kCVPixelBufferProResRAWKey_MetadataExtension not bound
 !missing-pinvoke! CVBufferCopyAttachment is not bound
 !missing-pinvoke! CVBufferCopyAttachments is not bound
+!missing-pinvoke! CVBufferHasAttachment is not bound
+## appended from unclassified file
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/iOS-CoreVideo.todo:7]:
```suggestion
```

**Review by rolfbjarne — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

File: tests/xtro-sharpie/MacCatalyst-CoreGraphics.todo [hunk 5]
```diff
@@ -106,9 +111,9 @@
 !missing-pinvoke! CGDisplayModeGetTypeID is not bound
 !missing-pinvoke! CGDisplayModeGetWidth is not bound
 !missing-pinvoke! CGDisplayModeIsUsableForDesktopGUI is not bound
-!missing-pinvoke! CGDisplayModelNumber is not bound
 !missing-pinvoke! CGDisplayModeRelease is not bound
 !missing-pinvoke! CGDisplayModeRetain is not bound
+!missing-pinvoke! CGDisplayModelNumber is not bound
```
Comment (spouliot) [inline tests/xtro-sharpie/MacCatalyst-CoreGraphics.todo:116]: not big deal - but it seems your sorting (Makefile) and the one done by C# differs wrt capital letters
and that might become noisy

**Review by spouliot — APPROVED** [review]

File: tests/xtro-sharpie/MacCatalyst-CoreGraphics.todo (hunk 5 above)
Comment (mandel-macaque) [reply tests/xtro-sharpie/MacCatalyst-CoreGraphics.todo:116]: I'll see if the sort command can take the same sorting algo.

**Comment by mandel-macaque** [comment]
No related test failure: https://github.com/xamarin/maccore/issues/2443

**Pull request merged**